index.html

```diff
@@ -611,92 +611,85 @@ <h5>ICT Development Training</h5>
+        <p>Qualified individuals with disabilities should be employed throughout an [=organization=]&apos;s hierarchy &#40;that is, all job types, all authority levels, and every department&#41; so that their unique insights and lived experiences can better inform decision-making.</p>
+        
+        <section id="evaluate-personnel">
+          <h4>How to Evaluate Support Maturity Level</h4>
+          <ol>
+            <li>Download the <a href="https://www.w3.org/TR/maturity-model/A11yMaturityTemplate.xlsx">maturity model spreadsheet</a>.</li>
+            <li>List all the organization&apos;s current Personnel efforts.</li>
+            <li>Compare the list to the spreadsheet to decide which proof points will be used to assess your organization&apos;s “Personnel” accessibility maturity. Not all proof points will be used for every business or organization. The proof points in section 3.5.2 are non-exhaustive examples of criteria.</li>
+          </ol>
+          <p>The level is <b>Inactive</b> when proof points demonstrate that:</p>
+            <ul>
+              <li>no effort has been made or only isolated efforts have been identified.</li>
+            </ul>
+          <p>The level is <b>Launch</b> when proof points demonstrate that:</p>
+            <ul>
+              <li>including employees with disabilities in the workforce has been recognized </li>
+              <li>targeted recruiting of qualified candidates with disabilities has been initiated, but recruitment, retention, engagement, and activities related to disability inclusion are not well-organized</li>
+              <li>accessible hiring announcements that encourage applications from the disability community are posted</li>
+              <li>equal employment opportunities for people with disabilities is specifically stated in company diversity and inclusion policies and statements</li>
+              <li>a champion has been designated to facilitate and mature disability inclusion</li>
+              <li>plans are in place for providing disability inclusion training, relevant to each individual’s position.</li>
+            </ul>
+          <p>The level is <b>Integrate</b> when proof points demonstrate that:</p>
+            <ul>
+              <li>a disability inclusion roadmap that drives ICT accessibility is in place</li>
+              <li>the overall organizational approach to evaluating recruitment, retention, advancement, and engagement is defined </li>
+              <li>process integration for maturing disability inclusion efforts for ICT accessibility is in progress but not consistently implemented across the organization</li>
+              <li>the company has identified strategic positions to employ people with disabilities who will help audit and drive the development of accessible products and services</li>
+              <li>targeted recruiting of employees with disabilities with an accessible recruiting process</li>
```

Comment:
Below you removed "targeted" should this be adjusted as well?  There are two mentions in the stages of "targeted".

Reply:
Rest looks good to me.